atlas/packages/app/src/components/Navbar/Navbar.tsx

```diff
@@ -1,60 +1,87 @@
 import React, { useState } from 'react'
-import { navigate, Link, RouteComponentProps } from '@reach/router'
+import { RouteComponentProps, Link, navigate } from '@reach/router'
 
-import routes from '@/config/routes'
 import { Icon } from '@/shared/components'
-import { Header, NavigationContainer, StyledIcon, StyledSearchbar, CancelButton, Logo } from './Navbar.style'
+import routes from '@/config/routes'
+import {
+  Header,
+  NavigationContainer,
+  StyledIcon,
+  StyledSearchbar,
+  SearchbarContainer,
+  CancelButton,
+  CancelButtonContainer,
+  Logo,
+} from './Navbar.style'
 
-const Navbar: React.FC<RouteComponentProps> = () => {
-  const [search, setSearch] = useState('')
-  const [isSearching, setIsSearching] = useState(false)
+type NavbarProps = RouteComponentProps
 
-  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    setSearch(e.currentTarget.value)
-  }
+const Navbar: React.FC<NavbarProps> = () => {
+  const [search, setSearch] = useState('')
+  const [focus, setFocus] = useState(false)
 
   const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
-    if (e.key === 'Enter' || (e.key === 'NumpadEnter' && search.trim() !== '')) {
+    if (e.key === 'Enter' || (e.key === 'NumpadEnter' && search.trim())) {
       navigate(routes.search(search))
     }
+    if (e.key === 'Escape' || e.key === 'Delete' || e.key === 'Esc') {
+      setFocus(false)
+      setSearch('')
+    }
+  }
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (search !== '' && e.currentTarget.value === '') {
+      setFocus(false)
```

Comment:
I was indecisive on this one, but I decided to pull it in, after reading your thoughts I am leaning on removing it. It could give some eyestrain since the searchbar is constantly jumping between expanded and not expanded,  for example if you mistype the first character of your query

Reply:
I think this still isn't changed?